deselected_tests.yaml

```diff
@@ -29,3 +29,6 @@ deselected_tests:
   # For Newton-CG solver, solution computed in float32 disagrees with that of float64 a little more than
   # the test expects, see https://github.com/scikit-learn/scikit-learn/pull/13645
   - linear_model/tests/test_logistic.py::test_dtype_match
+
+  # _hist_gradient_boosting tests (added in 0.21.0) times out in CircleCI
+  - ensemble/_hist_gradient_boosting/tests >=0.21.0
```

Comment:
Is any progress expected in reducing this list?

Reply:
This was added not because it was failing, but because build was timing out.

Per @ogrisel, this implementation is not finished yet (name starts with underscore).

Reply:
Actually it's shipped but in "experimental" mode meaning that the public API might change without going through a deprecation cycle in 0.22.

To avoid the timeout you can set `OMP_NUM_THREADS=2` in the circle ci build.

Reply:
Note that we have add a similar issue with pypy tests on our circle ci worker in https://github.com/scikit-learn/scikit-learn/pull/13800 (but the remaining memory usage issue is unrelated).